capplets/Makefile.am

```diff
@@ -1,3 +1,10 @@
+if HAVE_SYSTEMD
+TIMEADMIN = time-admin
+else
+TIMEADMIN =
```

Comment:
There's no real need to define `TIMEADMIN` empty in the false clause (undefined MAKE variables expand to an empty value), but I guess it doesn't hurt either.

Reply:
I used a more concise method